frontend/packages/db-structure/src/parser/schemarb/parser.ts

```diff
@@ -209,7 +203,14 @@ function extractColumnOptions(hashNode: KeywordHashNode, column: Column): void {
         column.notNull = value instanceof FalseNode
         break
       case 'default':
-        column.default = extractDefaultValue(value)
+        if (
+          value instanceof TrueNode ||
+          value instanceof FalseNode ||
+          value instanceof StringNode ||
+          value instanceof IntegerNode
+        ) {
+          column.default = extractDefaultValue(value)
+        }
```

Comment:
This was needed since `value` is a `Node`, which doesn’t match the argument type expected by `extractDefaultValue`.